.github/workflows/api-tests.yml

```diff
@@ -95,8 +95,8 @@ jobs:
               environment:
                 - PMM_DEBUG=1
                 - PERCONA_TEST_CHECKS_INTERVAL=10s
```

Comment:
did we change this env variable?

Reply:
Probably missed it because I couldn't find anywhere we use it. We can drop it?